src/main/java/com/scalar/database/transaction/consensuscommit/Coordinator.java

```diff
@@ -25,7 +25,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** */
+/**
+ * Keeps track of the {@link State} of a {@link Snapshot} or {@link Result} for {@link
+ * DistributedStorage}
```

Comment:
`coordinator.state` table has the status of each transaction like `COMMITTED` or `ABORTED`.
I think this class is used to get/put these statuses from/to `coordinator.state` table.

Reply:
Right. Thank you!